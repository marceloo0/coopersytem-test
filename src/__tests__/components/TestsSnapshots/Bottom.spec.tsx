import React from 'react';
import renderer from 'react-test-renderer';
import Buttom from '../../../components/Bottom';

test('renders correctly', () => {
  const bottom = renderer
    .create(<Buttom text="CONFIRMAR RESGATE" onPress={() => {}} />)
    .toJSON();
  expect(bottom).toMatchSnapshot();
});
