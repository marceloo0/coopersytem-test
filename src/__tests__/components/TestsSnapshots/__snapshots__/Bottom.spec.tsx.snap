// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<TouchableHighlight
  onPress={[Function]}
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#ffff00",
        "height": 60,
        "justifyContent": "center",
        "marginTop": 15,
        "width": "100%",
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "blue",
          "fontSize": 21,
          "fontWeight": "bold",
        },
      ]
    }
  >
    CONFIRMAR RESGATE
  </Text>
</TouchableHighlight>
`;
